C/Calcium/build_tarballs.jl

```diff
@@ -61,9 +61,9 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="FLINT_jll"), compat = "~200.700"),
-    Dependency(PackageSpec(name="Arb_jll"), compat = "~200.1900"),
-    Dependency(PackageSpec(name="Antic_jll"), compat = "~0.200"),
+    Dependency(PackageSpec(name="FLINT_jll"), compat = "~200.800"),
+    Dependency(PackageSpec(name="Arb_jll"), compat = "~200.2000"),
+    Dependency(PackageSpec(name="Antic_jll"), compat = "~0.200.500"),
```

Comment:
To be clear, this is not supposed to be compatible with 0.200.300 nor 0.200.400?

Reply:
Yes, I believe this is intended (right @fingolfin?)

Reply:
Yes, I checked and this is what I want.

Reply:
LGTM